### DISCUSSION TITLE
juicy/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#49] [Announcements]
# juicy/

REVIEWERS ARE ASLEEP POST DUMP TRUCKS

https://www.supergoodcode.com/juicy/

**Comment by Dark-Show** [comment]
![dump truck](https://kidsdreamstore.dk/images/list/cat.jpg)

**Comment by qwertychouskie** [comment]
https://www.youtube.com/watch?v=--X6-xUbuU0
